modules/storage.js

```diff
@@ -23,5 +24,15 @@ export class Store {
     books.push(book);
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  static removeBook(isbn) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (book.isbn === isbn) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
 }
```

Comment:
- As required by the previous reviewer, please ensure the deleted book is updated in the localStorage and the deleted book should be removed from the UI. As of now, when I reload your page, the deleted shows up again on your UI. Please, kindly fix this..